ang/test/mocks/data/award-additional-details.data.js

```diff
@@ -0,0 +1,12 @@
+(function () {
+  var module = angular.module('civiawards.data');
+
+  module.constant('AwardAdditionalDetailsData', {
```

Comment:
can we add to all of these a `MockData` suffix so we avoid collision with real services?